### PR TITLE
fixes and improvements

### PR DESCRIPTION
**Pull request by soundsnow**
I haven't had a good rest these days, I am staying up late to modify info files, and now I am very tired

The name of the developer or manufacturer is more than three words or names will be capitalized, such as: KONAMI and COLECO.  The logo itself is an abbreviation and is capitalized to be consistent, such as: DECO and MB
Epoch is changed to lower case because it can be looked up in the dictionary
Wikipedia: Titan (moon), the largest moon of Saturn
Mega Duck games will display its Logo before they starts, which is built into the ROM Cartridge.  I didn't find any games showing Cougar Boy's Logo. Mega Duck is its original name
Remove Cougar Boy

Replace display_name with abbreviations, they can be searched more easily, and are more suitable for small screens, and are more friendly to multi-system emulators
Add non-personal developers and manufacturers
Update some core versions
Remove parts of system and extensions for some cores, since the info file didn't mention the indispensable BIOS
Add new systems and extensions to the cores that provided the BIOS
Current does not need to be displayed, because other cores have years
Show NES Sound Format and Quick Disk for fixNES since only its extensions show NSF and QD
Two 3DOs have been duplicated, I changed one of them to its manufacturers
Use the highest model for the multisystem emulators, otherwise it will be very long
MSX/MSX2/MSX2+/MSXTurboR/SV/CV/SG/SC/SF
MSXTurboR/SV/CV/SF
SF-7000 requires SC-3000, and SC-3000 is compatible with SG-1000 games.
blueMSX machines I found have SVI-318 and SVI-328.  but on Wikipedia
They're not SVI, SV-318, SV-328
They're SVI, SVI-728, SVI-738
These are obvious typos, I have an SV but its logo is not SVI. This seems to be a mistake made by blueMSX developers
SVI is compatible with MSX, but SV is not compatible with MSX.  And their logos are different
NAOMI is a backronym for New Arcade Operation Machine Idea. Replace Naomi with NAOMI. It's arcade machine, it's not a Japanese given name like 直美,尚美,直己,尚己,直実,尚実,奈緒美
Replace Sony with SONY as I've never seen its logo in Sony
Fix typos
Fix bugs
Fix context dependencies
Add missing BIOS
Add new systems
Add new extensions
Remove non-working code
Channel F, short for "Channel Fun"
SEGA was SErvice GAmes
NAMCO was NAkamura Manufacturing COmpany
GoldStar was renamed to LG in 1995. 3DO: 1993–1997. New LG logo in late 3DO games
ZX81 isn't ZX 81 Wikipedia
If there are too many manufacturers, only its developers are kept.  For example: CD-i and MSX
Do not display the developer's name unless it or their name represents a company.  Such as HP: Hewlett-Packard (In fact its logo is hp not HP), MB: Milton Bradley
Nintendo released a miiiniii handheld console instead of a fake Mini.  It can be seen on the product packaging. new and amiibo also seem to be lowercase.  I put new and 3DS together for the convenience of searching, you don’t need to type spaces
Like you did with the CD-i, I removed the hyphen (minus) so it's easier to search
I really don't understand why some people just want their name to appear twice.  The only thing I didn't modify was NES because its successor is SNES
Nintendo - N64 (Nintendo - Nintendo 64), Commodore - C64 (Commodore - Commodore 64), Sega - SMS ( Sega - Sega Master System), Palm - Palm OS, Texas Instruments - TI-83 (Texas Instruments - Texas Instruments 83), DC Comics (Detective Comics Comics)
LD is LaserDisc
Remove extra decimal points and zeros in some versions
MAME is an Arcade emulator, so MESS is non-Arcade
mupen64plus next can only load two files (Disk+Cartridge) using the subsystem.  I can't directly make it load only one ndd file.  Seems like the developer didn't provide support, what a waste of time.  But even so, I added ndd to supported extensions
G7000 is Videopac, it is not Videopac+
Remove www. (emulator cores only) because it was too long and most sites can be accessed without typing https://www.
7800 EU BIOS
3DS Firmware/BIOS/Keys
DD IPL
SMS/GG BIOS
GB/GBC BIOS
There are others I can't remember.
I guess there are more missing or bugs you didn't fix.  I don't care about the rest, I've turned off notifications and won't reply to you.  My eyes are tired and my fingers are sore.
If you are dissatisfied with my modification, you can modify it yourself after merging, I may not be logged into my account for weeks.
other bugs I'm guessing may not have been fixed yet: typos, BIOS/Firmwares, systems, extensions, database, versions
Some seem to be copy-pasted, some don't seem to have been tested, or asked developers

### CODE REVIEW

**Comment by hizzlekizzle** [comment]
Since soundsnow isn't getting bothered by notifications for this thread (thanks for the PR, if you read this :) ), I think we can use it to discuss the impact a bit.

Overall, I like the changes. I think a few of the abbreviations--while more consistent--are a bit overly aggressive for brevity vs clarity (e.g., dropping Mega Duck to just MD doesn't really give would-be users enough info; googling 'MD retro game' is going to give you a bunch of Mega Drive hits). Likewise, I think 'N64' and 'C64' are needed to disambiguate those consoles from both being just '64'.

I would personally prefer 'Famicom' stay verbose, since that's what people call it, while nobody says 'Nintendo Entertainment System' but rather just 'NES'. Ditto for NeoGeo. Nobody calls it NG. Nobody calls Pico8 'P8', so that's likely to confuse, but adding 'Lexaloffle' in front is a great addition, I think. For that matter, I like adding more manufacturer/publisher names in front of single-purpose cores, as well, to help people understand what they're for.

I definitely agree with changing Naomi to NAOMI, but SEGA and SONY are both presented as Sega and Sony, respectively, on their own websites. I also think it's worthwhile to keep the notes on the NAOMI firmware for Flycast that call out that they are specifically for NAOMI, just so end-users know they don't need to hunt down these esoteric files just to play some Dreamcast games (speaking of Dreamcast, that's another one where I think it's worth keeping verbose vs DC, along with Playstation instead of PS, though PS2 was common enough that I think it's worth keeping short).

@RobLoach do you have any thoughts on this stuff, seeing as you handle a lot of our metadata? Of specific concern to your expertise, this PR removes the BSX and Satellaview databases references from the SNES cores. Is that good/appropriate?

File: citra_libretro.info [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Software Information
-display_name = "Nintendo - 3DS (Citra)"
+display_name = "Nintendo - 3DS/new3DS (Citra)"
```
Comment (RobLoach) [inline citra_libretro.info:2]: New 3DS?

File: fake08_libretro.info [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Software Information
-display_name = "Pico-8 (FAKE-08)"
+display_name = "Lexaloffle - P8 (FAKE-08)"
```
Comment (RobLoach) [inline fake08_libretro.info:2]: P8? It's PICO-8

File: gambatte_libretro.info [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Software Information
-display_name = "Nintendo - Game Boy / Color (Gambatte)"
+display_name = "Nintendo - GB/GBC (Gambatte)"
```
Comment (RobLoach) [inline gambatte_libretro.info:2]: I still believe acronyms are a bad idea here

**Review by RobLoach — CHANGES_REQUESTED** [review]

File: gambatte_libretro.info (hunk 3 above)
Comment (hizzlekizzle) [reply gambatte_libretro.info:2]: how would you feel about "Game Boy/GBC"? I feel like GBC and GBA are distinct and common enough (and also long enough) to be recognizable on their own.

File: citra_libretro.info (hunk 1 above)
Comment (hizzlekizzle) [reply citra_libretro.info:2]: I guess they're treating it as its own thing, like DSi, which has a certain logic to it, assuming there are n3DS-specific games (the SNES VC is one, I guess).

**Comment by hizzlekizzle** [comment]
Ok, I'll try to find some time to review it again in the next couple of days. Thanks for working so hard on this :)

**Pull request closed without merging**